packages/ringcentral-widgets-test/test/integration-test/calls/helper.js

```diff
@@ -15,7 +15,7 @@ import {
 } from '../../support/callHelper';
 
 function mockCallProcedure(func) {
-  return async function (phone, ...args) {
+  return async (phone, ...args) => {
```

Comment:
do we have to bind `this`?

Reply:
not, just because the eslint prompted it

Reply:
OK